pjsip/src/pjsua-lib/pjsua_call.c

```diff
@@ -954,6 +955,16 @@ PJ_DEF(pj_status_t) pjsua_call_make_call(pjsua_acc_id acc_id,
         goto on_error;
     }
 
+    /*
+     * Use pre defined Call-ID to be sent out with INVITE as opposed to using
+     * a randomly generated Call-ID
+     */
+
+    if( opt->custom_call_id.slen > 0 ){
+        dlg->call_id->id = opt->custom_call_id;
```

Comment:
Need to duplicate the string using pj_strdup() as `opt` argument is const.

Reply:
Ok I see - thank you for pointing that out.